kratos/python_scripts/save_restart_process.py

```diff
@@ -33,8 +37,8 @@ def __init__(self, model, params):
 
         model_part = model[params["model_part_name"].GetString()]
 
-        if model_part.GetCommunicator().TotalProcesses() > 1: # mpi-execution
-            from KratosMultiphysics.TrilinosApplication.trilinos_restart_utility import TrilinosRestartUtility as RestartUtility
+        if KratosMultiphysics.KratosGlobals.Kernel.IsDistributedRun(): # mpi-execution
```

Comment:
```suggestion
        if model_part.IsDistributed(): # mpi-execution
```
just bcs Kratos is executed in MPI does not mean that the ModelPart is dostributed

Reply:
Agreed, although this makes me think that we may have some problems when reading this same model part during a distributed run... How are we going to detect if it should be read by some or all processes?

Reply:
well this has to be set beforehand by whoever sets the mpi-communicator
We will have to review this again once we are able to split the comm